.github/workflows/bundler-integrations.yml

```diff
@@ -35,8 +35,17 @@ jobs:
         run: npm run ${{matrix.bundler}} -w @govuk-frontend/bundler-integrations
 
       # Check output for modules that should not be included
-      - name: Check output
+      - name: Check absence of unused modules in `single-component.js`
         working-directory: ./.github/workflows/bundler-integrations
         run: |
           ! grep "Accordion" dist/${{matrix.bundler}}/single-component.js -q
+
+      - name: Check absence of unused utility functions in `single-component.js`
+        working-directory: ./.github/workflows/bundler-integrations
+        run: |
+          ! grep "getFragmentFromUrl" dist/${{matrix.bundler}}/single-component.js -q
+
+      - name: Check presence of modules in `initAll.js`
+        working-directory: ./.github/workflows/bundler-integrations
+        run: |
```

Comment:
**note** Splitting those on their own command. Despite [GitHub using the -e flag for its default shell](https://github.com/github/docs/issues/23853), which should fail fast, the grep for `getFragmentFromUrl` failing didn't lead to the failure of the check.